src/wopiserver.py

```diff
@@ -25,9 +25,17 @@
     from werkzeug.exceptions import MethodNotAllowed as Flask_MethodNotAllowed
     import jwt                     # JSON Web Tokens support
     from prometheus_flask_exporter import PrometheusMetrics    # Prometheus support
+
 except ImportError:
     print("Missing modules, please install dependencies with `pip3 install -f requirements.txt`")
     raise
+
+try:
+    from waitress import serve
```

Comment:
could this block be moved to line 182 where the config is checked whether to run `waitress`?

Reply:
Sure.

I was actually thinking about if you make the waitress module optional. What I mean is: If it can not be imported, we do not exit with an error, but just write a warning and just do not have that functionality. 
Unfortunately I am not deep enough in python for this. @glpatcern do you know if that is possible?

Reply:
I was thinking the same :) and yes it is possible, but: if a site administrator has configured waitress to be enabled, shouldn't we loudly fail as opposed to run with flask? After all the wopiserver fails for any config entry that is incorrect/missing and for which there's no valid default.

At the same time, I'm now thinking to include the module in `requirements.txt` so that it is in the Docker image (it's tiny anyway) in all cases.